src/apps/replication/lib/replica_stub.cpp

```diff
@@ -393,6 +393,8 @@ void replica_stub::on_config_proposal(const configuration_update_request& propos
         return;
     }
 
+    // TODO(qinzuoyan): if all replicas are down, then the meta server will choose one to assign primary,
+    // if we open the replica with new_when_possible = true, then the old data will be cleared, is it reasonable?
```

Comment:
You are right, and we need to handle DDD more carefully. We probably want to have CT_LOAD_PRIMARY as one more proposal type to handle this case.